install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+org="puppetlabs"
+repo="pdkgo"
+
+app="pct"
+
+arch="x86_64"
+# os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+os="$(uname -s)"
+ext=".tar.gz"
+
+releases=$(curl -s -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/${org}/${repo}/releases")
+
+tag=$(echo $releases | grep -oE '"tag_name": "v([0-9]+\.)+[0-9]+' | grep -oE -m 1 'v([0-9]+\.)+[0-9]+')
+ver=$(echo $tag | sed 's/v//')
+file="${app}_${ver}_${os}_${arch}${ext}"
+
+downloadURL="https://github.com/${org}/${repo}/releases/download/${tag}/${file}"
+
+destination="${HOME}/.puppetlabs/pct"
+
+[ -d ${destination} ] || mkdir -p ${destination}
+
+echo "Downloading and extracting ${tag} to ${destination}"
+curl -L -s ${downloadURL} -o - | tar xz -C ${destination}
```

Comment:
Does this handle overwriting existing files?

Reply:
Good question - I had assumed so, but, just to be sure, double checked:
```sh
➜  pct ls -alt
total 14584
drwxr-xr-x  6 user  group      192 27 May 14:44 .
-rw-r--r--@ 1 user  group     6148 27 May 14:41 .DS_Store
drwxr-xr-x  8 user  group      256 27 May 14:40 ..
-rwxr-xr-x@ 1 user  group  7437696 25 Mar 17:38 prototype
-rw-r--r--@ 1 user  group    11357 25 Mar 17:36 LICENSE
-rw-r--r--@ 1 user  group     4311 25 Mar 17:36 README.md
```

Modified the [v0.2.0 release](https://github.com/sanfrancrisko/prototype/releases/tag/v0.2.0) to contain an updated `README`, then ran:
```
./install.sh
```
```sh
➜  pct ls -alt
total 15512
drwxr-xr-x@ 6 user  group      192 27 May 14:44 .
-rw-r--r--@ 1 user  group     6148 27 May 14:41 .DS_Store
drwxr-xr-x  8 user  group      256 27 May 14:40 ..
-rw-r--r--@ 1 user  group     4348 27 May 14:28 README.md
-rwxr-xr-x@ 1 user  group  7437696 25 Mar 17:38 prototype
-rw-r--r--@ 1 user  group    11357 25 Mar 17:36 LICENSE
```

... so we're all good 👍